src/frameworks/WebKit/src/WebView.m

```diff
@@ -0,0 +1,6 @@
+#import <WebKit/WebView.h>
+
+NSString * WebElementImageKey = @"WebElementImage";
+NSString * WebElementLinkURLKey = @"WebElementLinkURL";
+NSString * WebElementLinkTargetFrameKey = @"WebElementLinkTargetFrame";
+NSString * WebElementLinkTitleKey = @"WebElementLinkTitle";
```

Comment:
Don't add spaces between the asterisk, it should be either be `NSString *WebElementImageKey` or `NSString* WebElementImageKey`

```suggestion
NSString *WebElementImageKey = @"WebElementImage";
NSString *WebElementLinkURLKey = @"WebElementLinkURL";
NSString *WebElementLinkTargetFrameKey = @"WebElementTargetFrame";
NSString *WebElementLinkTitleKey = @"WebElementLinkTitle";
```